spec/project_spec.rb

```diff
@@ -84,7 +84,9 @@
     it 'has link definitions for all implicit links' do
       implicit_link_names = changelog.scan(/\[([^\]]+)\]\[\]/).flatten.uniq
       implicit_link_names.each do |name|
-        expect(changelog).to include("[#{name}]: http")
+        expect(changelog.include?("[#{name}]: http"))
+          .to be_truthy, "CHANGELOG.md is missing a link for #{name}. " \
```

Comment:
Good improvement 👍 
This is small feedback. Since `be_truthy` treats `'str'`, `1`, and so on as true, I think that `be true` is preferable for behaviors that return boolean value.

Reply:
Thanks, @koic .

I'm aware of this, and that's actually what I tried at first.
However, as far as I can tell, `be` is a type of _operator matcher_, and therefore you cannot supply a custom error message with it.

And in this case, although I agree that `be true` would be a little more accurate, the end result is the same.

Reply:
My explanation was not enough. A custom message is an argument of `to` method. 

```ruby
def to(matcher=nil, message=nil, &block)
```

https://github.com/rspec/rspec-expectations/blob/v3.8.2/lib/rspec/expectations/expectation_target.rb#L63

When a custom message is given, parentheses can not be omitted for `be(true)`.

```diff
 expect(changelog.include?("[#{name}]: http"))
-          .to be_truthy, "CHANGELOG.md is missing a link for #{name}. " \
+          .to be(true), "CHANGELOG.md is missing a link for #{name}. " \
```

Reply:
Ahhh, that's what I was missing! Ruby can be funny about brackets sometimes 👀 

Fair enough, I'll update as you suggest.

Reply:
Looks good. Can you squash your commits into one?